### ISSUE TITLE
Mutation with custom GraphQLInputObjectType

### ISSUE THREAD

**Issue by carleryd** [jahewson/elm-graphql#10]
I'm trying to generate a mutation with a custom `GraphQLInputObjectType` which works in GraphiQL but not when running the script. Generation works fine when using `String!`.
This is the mutation I'm trying to generate
```
mutation CreateLinkMutation($mutateParam: CreateLinkInput!) {
  createLink(input: $mutateParam) {
    clientMutationId,
    linkEdge {
      cursor
      node {
        createdAt
        id
        title
        url
      }
    }
  }
}
```
and I'm using this input in GraphiQL
```
{
  "mutateParam": {
    "url": "http://test.com",
    "title": "Test1",
    "clientMutationId": "0"
  }
}
```
Is this not yet supported for Mutations? Is it on the road map? I manually edited the generated from elm-graphql code in 0.16 to achieve this but that is quite inflexible :)

**Comment by jahewson** [comment]
Looks like I didn't implement support for records as input values. Will fix that soon.

**Comment by note89** [comment]
+1

**Comment by proProbe** [comment]
+1

**Comment by rgabrielsson** [comment]
+1

**Comment by matslexell** [comment]
+1

**Comment by MartinHell** [comment]
+1

**Comment by jahewson** [comment]
This feature has landed in e63d966e025036fb7397e574ff07fe74db402c82.

**Comment by note89** [comment]
Cool ! tnx a lot ! :)

**Comment by carleryd** [comment]
Great job jahewson, really needed this feature! Thanks!!!

**Comment by jahewson** [comment]
Some bug fixes landed in 2234a433d44b4c7bdb412abb738f45b0077a8ea3 too.
